geronimo-opentracing-common/src/main/java/org/apache/geronimo/microprofile/opentracing/common/microprofile/zipkin/ZipkinConverter.java

```diff
@@ -50,6 +50,7 @@
     private IdGenerator idGenerator;
 
     private String serviceName;
+    private boolean useV2 = false;
```

Comment:
I think it can be worth to have 2 converters and configure it in the extension as other observers. Will ensure setup is not resolved at runtime.

Wdyt?